### PR TITLE
New bru2nii interface and example data structure

### PR DESCRIPTION
**Pull request by TheChymera**
Updated #1353

### CODE REVIEW

**Comment by TheChymera** [comment]
Can you help me figure out what's wrong with travis? I don't see how the error seen in [the log](https://s3.amazonaws.com/archive.travis-ci.org/jobs/136163442/log.txt) relates to the interface.

**Comment by TheChymera** [comment]
@satra @chrisfilo 

I can't satisfy both travis and circle at the same time:

This is what I get now from travis:

```
File "/home/travis/build/nipy/nipype/nipype/interfaces/bru2nii.py", line 35, in nipype.interfaces.bru2nii.Bru2
Failed example:
    converter.cmdline
Expected:
    'Bru2 -o /home/ubuntu/nipype/nipype/testing/data/brukerdir brukerdir'
Got:
    'Bru2 -o /home/travis/build/nipy/nipype/nipype/testing/data/brukerdir brukerdir'
```

And this is what I used to get from circle before I tried to "fix" it:

```
File "/home/travis/build/nipy/nipype/nipype/interfaces/bru2nii.py", line 35, in nipype.interfaces.bru2nii.Bru2
Failed example:
    converter.cmdline
Expected:
    'Bru2 -o /home/travis/build/nipy/nipype/nipype/testing/data/brukerdir brukerdir'
Got:
    'Bru2 -o /home/ubuntu/nipype/nipype/testing/data/brukerdir brukerdir'
```

Can you help me figure out how to fix this?

File: nipype/interfaces/bru2nii.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		if isdefined(self.inputs.output_filename):
+			output_filename1 = self.inputs.output_filename
+		else:
+			output_filename1 = self._gen_filename('output_filename')
+		outputs["nii_file"] = output_filename1+".nii"
+		return outputs
+
+	def _gen_filename(self, name):
+		if name == 'output_filename':
+			outfile = os.getcwd()+"/"+os.path.basename(os.path.normpath(self.inputs.input_dir))
```
Comment (chrisfilo) [inline nipype/interfaces/bru2nii.py]: Please replace by os.path.join

File: nipype/interfaces/bru2nii.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	"""Uses bru2nii's Bru2 to convert Bruker files
+
+	Examples
+	========
+
+	>>> from nipype.interfaces.bru2nii import Bru2
+	>>> converter = Bru2()
+	>>> converter.inputs.input_dir = "brukerdir"
+	>>> converter.cmdline
+	'Bru2 -o /home/ubuntu/nipype/nipype/testing/data/brukerdir brukerdir'
```
Comment (chrisfilo) [inline nipype/interfaces/bru2nii.py]: To deal with absolute path you can use ELLIPSIS like here: https://github.com/nipy/nipype/blob/c96b3b12bd5dbc305aa9740326ff0cdaae16d0c2/nipype/interfaces/ants/registration.py#L504

See: https://pymotw.com/2/doctest/#handling-unpredictable-output

**Comment by satra** [comment]
@TheChymera - merge with master and push.

**Comment by TheChymera** [comment]
@satra could you please pul?

**Comment by satra** [comment]
@TheChymera - currently has conflicts.

**Pull request closed without merging**